src/utils/browserHistory.js

```diff
@@ -0,0 +1,6 @@
+import {useRouterHistory} from 'react-router';
+import {createHistory} from 'history';
```

Comment:
also docs suggest that it should be imported this way:
`import createHistory from 'history/createBrowserHistory'`
how does it work?